web/frontend/src/components/slam.vue

```diff
@@ -169,17 +191,30 @@ const executeMoveOnMap = async () => {
     new grpc.Metadata(),
     (error: ServiceError | null, response: motionApi.MoveOnMapResponse | null) => {
       if (error) {
-        moveClick = !moveClick;
+        allowMove = !allowMove;
+        executeDeleteDestinationMarker();
         toast.error(`Error moving: ${error}`);
         return;
       }
-      moveClick = !moveClick;
+      allowMove = !allowMove;
+      executeDeleteDestinationMarker();
       toast.success(`MoveOnMap success: ${response!.getSuccess()}`);
     }
   );
 
 };
 
+const executeStopMoveOnMap = () => {
+  for (const element of props.operations) {
+    if (element.op.method.includes('MoveOnMap')) {
```

Comment:
Each SLAM service is connected to a single base. If two different slam services are operating on two different bases, the Stopping of one will trigger the stopping of the other based on this current logic. Can we instead look to only send the `cancelOperation` command to element.op.id's associated with the particular base being stopped?

Reply:
> Each SLAM service is connected to a single base

I disagree. 
Suppose we have a robot with a base and lidar camera. However, the lidar is not mounted on the base and we instead create the slam map just by walking around.

There is no information in the config which tells up that a base is using a slam or vice versa.

Reply:
What is the ticket that was created for this point?

Reply:
https://viam.atlassian.net/browse/PRODUCT-481